config/_default/config.toml

```diff
@@ -199,9 +199,9 @@ weight = 4
 
 [[languages.en.menu.main]]
 identifier = "acknowledgments"
-name = "Ack"
+name = "Support"
```

Comment:
Why Support and not Acknowledgements?

Reply:
Is support not potentially confusing if one is looking for user-support?

Reply:
For clarity, "Sponsors" is probably the better choice.   I can fix up the body of the text to make it non-redundant and clear.  Also, there are other sponsors, and I'll say something like, "Sponsors include ..."

Reply:
Sure change it to Sponsors then